public/partials/shortcodes/process/process_form_submission.php

```diff
@@ -167,20 +167,20 @@
 
 // If this member does not exist, then we need to add the status_if_new flag and set our $new_subscriber variable
 // Likewise, if this member exists but their status is 'pending' it means we're dealing with a double opt-in list and they never confirmed
-// Or, if this member but their status is 'unsubscribed' it means we're dealing with someone who unsubscribed and they need to re-subscribe
+// Or, if this member exists but their status is 'unsubscribed' it means we're dealing with someone who unsubscribed and they need to re-subscribe (this could be single opt-in or double opt-in)
 // Continue as if they're a new member to force another double opt-in email
-$double_optin_resubscribe = is_array( $member_exists ) && isset( $member_exists['status'] ) && ( $member_exists['status'] === 'pending' || $member_exists['status'] === 'unsubscribed' ) ? true : false;
+$was_subscribed = is_array( $member_exists ) && isset( $member_exists['status'] ) && ( $member_exists['status'] === 'pending' || $member_exists['status'] === 'unsubscribed' ) ? true : false;
```

Comment:
For this line, you can drop `is_array( $member_exists )`, and you can also drop the `? true : false` from the end. The various comparison operators (`&&`, `||`, etc) return a boolean value natively.

Reply:
Good point. Thank you.